### PR TITLE
Speedup 1

### PR DESCRIPTION
**Pull request by IhorNehrutsa**
Several simple changes.
Not all faster, but it is more explicit.

### CODE REVIEW

File: src/hardware/motor.cpp [hunk 1]
```diff
@@ -15,8 +15,8 @@ StepperMotor::StepperMotor() {
     pinMode(COIL_POWER_OUTPUT_PINS[B], OUTPUT);
 
     // Configure the PWM current output pins
-    this -> PWMCurrentPinInfo[A] = analogSetup(COIL_POWER_OUTPUT_PINS[A], MOTOR_PWM_FREQ * 1000, 0);
-    this -> PWMCurrentPinInfo[B] = analogSetup(COIL_POWER_OUTPUT_PINS[B], MOTOR_PWM_FREQ * 1000, 0);
```
Comment (IhorNehrutsa) [inline src/hardware/motor.cpp:19]: ```* 1000```
is not clear

File: src/hardware/motor.cpp [hunk 2]
```diff
@@ -227,7 +227,7 @@ float StepperMotor::getMicrostepMultiplier() const {
 void StepperMotor::step(STEP_DIR dir, bool useMultiplier, bool updateDesiredAngle) {
 
     // Main angle change (any inversions * angle of microstep)
-    float angleChange = (this -> fullStepAngle) / (this -> microstepDivisor);
```
Comment (IhorNehrutsa) [inline src/hardware/motor.cpp:230]: It is an dispensable division

File: src/hardware/timers.cpp [hunk 3]
```diff
@@ -69,13 +69,13 @@ void setupMotorTimers() {
 // Pauses the timers, essentially disabling them for the time being
 void disableInterrupts() {
 
-    // Add one to the interrupt block counter
-    interruptBlockCount++;
-
     // Disable the interrupts if this is the first block
-    if (interruptBlockCount == 1) {
+    if (interruptBlockCount == 0) {
```
Comment (IhorNehrutsa) [inline src/hardware/timers.cpp:73]: The comparison with 0 is faster.

File: src/hardware/motor.cpp (hunk 2 above)
Comment (CAP1Sup) [reply src/hardware/motor.cpp:230]: Absolutely. The best part is that I wrote in support for it, just never actually implemented it :)

File: src/hardware/timers.cpp (hunk 3 above)
Comment (CAP1Sup) [reply src/hardware/timers.cpp:73]: Sounds good to me

**Comment by CAP1Sup** [comment]
Excellent job, this should help to make the code more clear and run just a bit faster

**Pull request merged**